src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -590,13 +590,13 @@ Valid from:</comment>
     <value>The primary signature and repository countersignature are unrelated.</value>
   </data>
   <data name="Error_NoRepoAllowList" xml:space="preserve">
-    <value>A repository announced that their packages should be signed but an empty list of trusted certificates was found.</value>
+    <value>The repository where this package was downloaded announced that their packages should be signed but an empty list of trusted signers was found. No package downloaded from this repository can be trusted.</value>
```

Comment:
Do we really want the `No package downloaded from this repository can be trusted`
Isn't this the case where you are not trusting any repositories?

Reply:
This is the case where a package said that all their packages are signed, but didn't give a list of trusted signers. I can remove that part of the message, I just wanted to add something that clearly states to the user why we are failing, since they don't have control over what the server is sending.

Reply:
I see. 
It's fine then, I just thought that the last part was not clear enough. 

Not sure if a customer will understand it.

Reply:
Any suggestion on how to improve this so it's easier for the customer to understand?

Reply:
`The repository this package was downloaded from announced that their packages are signed but an empty list of trusted signers was found. This is likely an issue with the repository.`